.gitignore

```diff
@@ -1,7 +1,8 @@
 /.dapper
 /.cache
 /bin
-/cmd/helm-project-operator/fs/project-operator-example.tgz.base64
+!/cmd/helm-project-operator/fs/.gitkeep
```

Comment:
why do we need to keep this dir? Can we recreated it when we need it?

Reply:
Yeah - that's a fair point.
I tend to favor keeping dirs like this around - feels like it keeps things uniform and gives hints to the fact that the dir will be populated as part of a successful build. Not super attached to that though if we think it should just be fully excluded.